roles/falcon_installation/defaults/main.yml

```diff
@@ -22,11 +22,10 @@ falcon_provisioning_token:
 #
 falcon_install_method: api
 
-# Specify GPG Key file location if the file exists in a different location or
-# the installation method is repo. This is purely an optional step but is
-# considered a best practice.
+# Specify whether or not to verify the Falcon sensor RPM based packages.
+# This is an optional step but is considered a best practice.
```

Comment:
Please reword this with some like:

`By default, the gpg key is installed.... Optionally.....`

Reply:
Updated